console/console-init/ui/src/Tests/AddressSapceHeader.test.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import {
+  IAddressSpace,
+  AddressSpaceHeader,
+} from 'src/Components/AddressSpace/AddressSpaceHeader';
+import { render } from '@testing-library/react';
+
+describe('Address Space Detail', () => {
+  test('it renders address space headers at top', () => {
+    const props: IAddressSpace = {
+      name: 'jBoss',
+      namespace: 'devops_jbosstest1',
+      createdOn: '2 days ago',
+      type: 'Brokered',
+      onDownload: () => {},
+      onDelete: () => {},
+    };
+
+    const { getByText } = render(<AddressSpaceHeader {...props} />);
+
+    const nameNode = getByText(props.name);
+    const namespaceNode = getByText(props.namespace);
+    const typeNode= getByText(props.type);
+    const createdOnNode = getByText(props.createdOn);
+
+    expect(nameNode).toBeDefined();
```

Comment:
no need to check for this explicitly, all the `get*` methods will throw an error if the searched content is not found.

Reply:
The same comment applies to the rest of the changes of course.

Reply:
Okay, Then what exactly do we need to test in these scenarios ?